pkg/kvstore/etcd.go

```diff
@@ -439,6 +434,47 @@ func (e *EtcdClient) DeleteTree(path string) error {
 	return err
 }
 
+// StartWatch starts watching for changes in a prefix
+func (e *EtcdClient) StartWatch(w *Watcher) {
+	go func(w *Watcher) {
```

Comment:
The function parameter is not necessary. This closure can refer to the w parameter.